README.md

```diff
@@ -2,42 +2,41 @@
 
 [![License: CC BY-SA 4.0](https://img.shields.io/badge/License-CC%20BY--SA%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-sa/4.0/)
 
-The Frameworkless Movement is a group of developers interested in developing applications without frameworks. We don't hate frameworks, nor we will ever create campaigns against frameworks, but we perceive the **abuse of the use of frameworks as a lack of knowledge regarding technical debt** and the availability of alternatives given by the vanilla language or by dedicated libraries.
+The Frameworkless Movement is a group of developers interested in developing applications without frameworks. We don't hate frameworks, nor we will ever create campaigns against frameworks, but we perceive the **abuse of frameworks as a lack of knowledge regarding technical debt** and the availability of alternatives given by the vanilla language or by dedicated libraries.
 
 ## Why the Movement
 
-Everytime that a team is using a framework, it is also taking a **risk**. This risk is, after some time, to be locked up with a tool that is not giving the team any kind of value and that most of the time it represent a major roadblock to change. Most importantly a framework could "die" way before the software that uses it, leaving the developers with a heavy burden.
+Every time a team uses a framework, it also takes a **risk**. The risk is that after some time has passed the team ends up with a tool that does not provide any kind of value anymore and that most of the times represents a major roadblock to change. Most importantly a framework could "die" way before the software that uses it, leaving the developers with a heavy burden.
```

Comment:
@GeoDoo why did you remove `locked`? I think it gives quite the idea...it also resemble the [Vendor Lock-in](https://en.wikipedia.org/wiki/Vendor_lock-in) anti-pattern.

Reply:
So, in the end i changed the meaning a lil bit meaning that you are eventually left to deal with a deprecated tool! I do not know if you want to keep it or change it back to lock up (maybe we can change lock up => lock in) I dunno for sure

:)

Reply:
@lorenzomassacci @adellava @Violo George convinced me here. What do you think?

Reply:
ok for me! thx @GeoDoo

Reply:
Ok for me too! Great @GeoDoo